include/numeric.h

```diff
@@ -97,7 +97,18 @@ namespace Numerics
 			: Numeric(ERR_NOSUCHNICK)
 		{
 			push(nick);
-			push("No such nick/channel");
+			push("No such nick");
+		}
+	};
+
+	class NoSuchChannel : public Numeric::Numeric
```

Comment:
Can you add documentation for this class? Something like this:

```
/** Builder for the ERR_NOSUCHNICK numeric. */
```

Reply:
Added that comment for both NoSuchNick and NoSuchChannel. Thanks!